### PR TITLE
Added support for satellite graphs

### PR DESCRIPTION
**Pull request by hkernbach**
Also added two new graph (cluster) option parameters:
- replicationFactor
- writeConcern

See: https://www.arangodb.com/docs/stable/http/gharial-management.html#create-a-graph

Simple example:
```
from pyArango.connection import *
from pyArango.collection import Collection, Edges, Field
from pyArango.graph import Graph, EdgeDefinition

databaseName = "satellite_graph_db"

conn = Connection()

# Cleanup (if needed)
try:
    conn.createDatabase(name=databaseName)
except Exception:
    pass

# Select our "satellite_graph_db" database
db = conn[databaseName] # all databases are loaded automatically into the connection and are accessible in this fashion

# Define our vertex to use
class Humans(Collection):
    _fields = {
        "name": Field()
    }

# Define our edge to use
class Friend(Edges):
    _fields = {
        "lifetime": Field()
    }

# Here's how you define a Satellite Graph
class MySatelliteGraph(Graph) :
    _edgeDefinitions = [EdgeDefinition("Friend", fromCollections=["Humans"], toCollections=["Humans"])]
    _orphanedCollections = []

theSatelliteGraph = db.createSatelliteGraph("MySatelliteGraph")
```

### CODE REVIEW

**Comment by joerg84** [comment]
@hkernbach Do you want to add the example from the description to the readme or is that too much detail?

**Comment by hkernbach** [comment]
@joerg84 I think that would be nice, but keep in mind that 3.7 is not released yet

File: pyArango/database.py [hunk 1]
```diff
@@ -195,6 +199,9 @@ def _checkCollectionList(lst):
             raise CreationError(data["errorMessage"], data)
         return self.graphs[name]
 
+    def createSatelliteGraph(self, name, createCollections = True, writeConcern = None):
+        return self.createGraph(name, createCollections, False, None, None, "satellite", None);
```
Comment (tariqdaouda) [inline pyArango/database.py]: writeConcern will be None regardless of the value entered as argument. I would also add the names of arguments explicitly as it makes it easier to spot these kind of bugs.

**Comment by hkernbach** [comment]
> Besides the `writeConcern` fixed at None, I don't see anything wrong with this PR. It's pretty cool.

Ups, writeConcern can be fully removed as a parameter in the new function. It will be set internally by the server automatically. The parameter will be ignored anyway.

**Comment by joerg84** [comment]
@tariqdaouda Can we merge this?

**Pull request merged**